src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -81,6 +90,64 @@ export const TransactionDetailsScene = (props: Props) => {
 
   const { name = '' } = localMetadata
 
+  // #region Crypto Fiat Rows
```

Comment:
This is a good cleanup! The only reason `TransactionFiatTiles` existed was so we could use hooks, and now that the parent is finally converted, all this stuff can finally move over.